pkg/okteto/kubetoken/store.go

```diff
@@ -0,0 +1,52 @@
+package kubetoken
+
+import (
+	"errors"
+	"fmt"
+	"os"
+)
+
+func NewFileByteStore(fileName string) *FileByteStore {
+	return &FileByteStore{
+		FileName:   fileName,
+		osStat:     os.Stat,
+		osCreate:   os.Create,
+		osReadFile: os.ReadFile,
+		writeFile: func(filename string, data []byte) error {
+			return os.WriteFile(filename, data, 0600)
+		},
+	}
+}
+
+type FileByteStore struct {
+	FileName   string
```

Comment:
Just a recommendation. We use `afero` to mock all the OS calls. Check it out because you will just need one field

Reply:
I've been trying to migrate to `afero` but I had an issue. How do I mock a call to give me an error? Like for example, I want to test if `os.Stat` returns an error